backend/Tests/Api/GraphQL/Import/TransactionsWriterTest.cs

```diff
@@ -612,6 +612,29 @@ public async Task TransactionEvents_ContractResumed()
         result.Success.Should().BeTrue();
     }
 
+
+    [Fact]
+    public async Task TransactionEvents_ContractUpgraded()
+    {
+        _blockSummaryBuilder
+            .WithTransactionSummaries(new TransactionSummaryBuilder()
+                .WithResult(new TransactionSuccessResultBuilder()
+                    .WithEvents(new Upgraded(
+                        new ContractAddress(1423, 1),
+                        new ModuleRef("73ba390d9ce2bb1bf54f124bb00e9dee0d6dc40d6de0f5ba06e1d1f095e4afcc"),
+                        new ModuleRef("73ba390d9ce2bb1bf54f124bb00e9dee0d6dc40d6de0f5ba06e1d1f095e4afcc")
```

Comment:
I recommend using a different hash for from and to in the test. Just 
```
aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
```
for example.

Reply:
done